L1Trigger/RegionalCaloTrigger/plugins/MaskedRctInputDigiProducer.cc

```diff
@@ -272,13 +272,13 @@ MaskedRctInputDigiProducer::produce(edm::Event& iEvent, const edm::EventSetup& i
 	     {
 	       //std::cout << "eta-: mask is " << ecalMask.at(0).at(iphi-1).at(absIeta-1) << std::endl;
 	       energy = ecalMask.at(0).at(iphi-1).at(absIeta-1) * ecalColl[i].sample(nSample).compressedEt();
-	       fineGrain = ecalMask.at(0).at(iphi-1).at(absIeta-1) * ecalColl[i].sample(nSample).fineGrain();
+	       fineGrain = ( ecalMask.at(0).at(iphi-1).at(absIeta-1) * ecalColl[i].sample(nSample).fineGrain() ) != 0;
```

Comment:
hi @mrodozov - i think the right fix is 

fineGrain = ( ecalMask.at(0).at(iphi-1).at(absIeta-1) !=0 ) &&  ecalColl[i].sample(nSample).fineGrain() 

[fineGrain() returns a bool]